cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -102,12 +102,26 @@ public void createCollection(String dbName, String collectionName, long dataExpi
         BasicDBObject keys;
         IndexOptions options;
         try {
```

Comment:
From which sink is this `createCollection()` is being called? MongoSink or STHSink?

Reply:
`createCollection(String dbName, String collectionName, long dataExpiration)` is being called from [STHSink](https://github.com/telefonicaid/fiware-cygnus/blob/master/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISTHSink.java#L376)